eultheme/templates/eultheme/site_base.html

```diff
@@ -42,13 +42,16 @@
     {% block head_extras %}{% endblock %}  {# any header metadata, etc #}
   </head>
   <body class="{% block body_class %}{% endblock %}"{% block body_attrs %}{% endblock %}>
-      {% if ENABLE_BETA_WARNING %}
-        <div class="ribbon beta-warning fade">
-          <h2>This site is in <code>BETA</code></h2>
-          <p>Content may change or be removed without warning.</p>
-          <button class="btn btn-default">OK</button>
-        </div>
-      {% endif %}
+    {% if ENABLE_BETA_WARNING %}
```

Comment:
Not related to your current changes, but I wonder if this should be a snippet also?